eng/common/scripts/copy-docs-to-blobstorage.ps1

```diff
@@ -90,23 +90,23 @@ function Sort-Versions
     $versionsObject.SortedVersionArray = @(SortSemVersions -versions ($VersionArray | % { ToSemVer $_}))
     $versionsObject.RawVersionsList = $versionsObject.SortedVersionArray | % { $_.RawVersion }
 
-    # handle latest and preview
-    # we only want to hold onto the latest preview if its NEWER than the latest GA.
-    # this means that the latest preview package either A) has to be the latest value in the VersionArray
+    # handle latest and beta
```

Comment:
Lets avoid making this change under the `eng/common` directory as that has a special update workflow. @chidozieononiwu can you make these updates with your other changes to this file.